docs/process/release-planning.md

```diff
@@ -1,7 +1,73 @@
 # Release Planning
 
-We plan work at two levels of granularity: first at the marketing release level (1.1, 1.2, etc.) and then at the milestone level. Marketing releases are made up of multiple milestones. Milestones should contain roughly a month’s worth of work.
+This document outlines our process for planning and scheduling releases, so you
+can familiarize yourself with the flow of work from opening an issue to seeing
+it published in a release.
 
-We’ll release updates as needed or when a milestone is completed.
+## Releases
 
-Large features and user-visible changes are feature-flagged, to be unflagged in the next marketing release. Beta and dev builds will have all feature flags enabled.
+We organize releases in two ways - marketing and milestones:
+
+ - Marketing releases are what we use to represent planned features, and are
+   high-level goals
+      - **e.g. 1.4, 1.5, etc**
+ - Milestones releases are used to track issues and pull requests associated
+   with a marketing release, and can be followed along on GitHub
+      - **e.g. 1.4.1, 1.4.2, 1.5.0, etc**
+
+We aim to ship updates to production every two weeks, to ensure a continuous
+flow of improvements to our users. You can follow along with the latest
```

Comment:
`Track our progress in the [current milestones](https://github.com/desktop/desktop/milestones?direction=desc&sort=completeness&state=open).`

I just hate the `here` links.